docs/workflows/public_data_sharing/retrieve_srr_metadata.md

```diff
@@ -0,0 +1,52 @@
+# Retrieve SRR Metadata Workflow
+
+## Quick Facts
+
+| **Workflow Type** | **Applicable Kingdom** | **Last Known Changes** | **Command-line Compatibility** | **Workflow Level** |
+|---|---|---|---|---|
+| [Public Data Sharing](../../workflows_overview/workflows_type.md/#public-data-sharing) | [Any Taxa](../../workflows_overview/workflows_kingdom.md/#any-taxa) | PHB v2.3.0 | Yes | Sample-level |
+
+## Retrieve SRR Metadata
+
+This workflow is designed to retrieve the Sequence Read Archive (SRA) accession (SRR) associated with a given sample accession. The primary inputs are BioSample IDs (e.g., SAMN00000000) or SRA Experiment IDs (e.g., SRX000000), which link to sequencing data in the SRA repository.
```

Comment:
I think based on this description this workflow should be renamed to indicate it only returns the ACCESSION, not any additional metadata.

Can you rename this workflow and everything associated with it -- file names, WDL code YAML entries, documentation, etc. -- to Fetch_SRR_Accession or something like that? that way the the name very clearly describes that it only returns the SRR accession instead of the mysterious metadata which could mean anything